nginx/Dockerfile

```diff
@@ -27,7 +27,7 @@ ARG INSTALLDIR=${BASEDIR}/nginx
 ARG SIG_ALG="dilithium3"
 
 # defines the list of default groups to be activated in nginx-openssl config:
-ARG DEFAULT_GROUPS=x25519:x448:kyber512:p256_kyber512:kyber768:p384_kyber768:kyber1024:p521_kyber1024
+ARG DEFAULT_GROUPS=x25519:x448:prime256v1:secp384r1:secp521r1:kyber512:x25519_kyber768:p256_kyber512:kyber768:p384_kyber768:kyber1024:p521_kyber1024:mlkem512:mlkem768:mlkem1024:X25519MLKEM768:SecP256r1MLKEM768
```

Comment:
An opportunity to remove kyber and dilithium? This will not work when the next oqsprovider release is done.

Reply:
Thanks for the suggestion @baentsch, I agree. Updated accordingly.